apps/agreement/test/agreement/agreement_new_action.js

```diff
@@ -211,6 +212,13 @@ contract('Agreement', ([_, owner, submitter, someone]) => {
                               const currentCashierBalance = await collateralToken.balanceOf(aragonAppFeesCashier.address)
                               assertBn(currentCashierBalance, previousCashierBalance.add(appFeeAmount), 'cashier balance does not match')
                             })
+
+                            it('references the agreement action ID in the cashier', async () => {
+                              const { actionId, receipt } = await disputable.newAction({ submitter, actionContext, stake, sign })
+
+                              assertAmountOfEvents(receipt, APP_FEES_CASHIER_EVENTS.APP_FEE_PAID, { decodeForAbi: aragonAppFeesCashier.abi })
+                              assertEvent(receipt, APP_FEES_CASHIER_EVENTS.APP_FEE_PAID, { expectedArgs: { by: disputable.agreement.address, appId: (await disputable.appId()), data: padLeft(toHex(actionId), 64) }, decodeForAbi: aragonAppFeesCashier.abi })
```

Comment:
Not sure how useful this test is, since we're considering the cashier to be a mock.

Reply:
it helps us to make sure we are forwarding that parameter correctly